apps/ecs/views/check-requested.html

```diff
@@ -8,5 +8,12 @@ <h1 class="govuk-panel__title">
         {{#t}}pages.check-requested.confirmed{{/t}}
       </h1>
     </div>
+    <p class="govuk-body">{{#t}}pages.check-requested.p1{{/t}}</p>
+    <div>
+      <button class="govuk-button request-another-check">
```

Comment:
the behaviour of this button is not clear, is it covered separately?

Reply:
I think because the session is cleared this button just redirects the user back to the start page /eligibility - is there a way I can make this more clear?

Reply:
This will submit the form, I would suggest using link instead to avoid the unnecessary steps associated with POST request.
```
    <a href="/eligibility" role="button" draggable="false" class="govuk-button" data-module="govuk-button">
      {{#t}}buttons.request-another-check{{/t}}
    </a>
```